themes/openy_themes/openy_rose/templates/paragraph/paragraph--teaser--default.html.twig

```diff
@@ -50,6 +50,8 @@ set classes = [
   <div class="col-sm-7 content">
     {{ content.field_prgf_title }}
     {{ content.field_prgf_description }}
-    <a href="{{ content.field_prgf_link.0['#url'] }}" class="btn btn-link blue">{{ content.field_prgf_link.0['#title'] }}</a>
+    {% if paragraph.field_prgf_link.0.uri %}
```

Comment:
It is safe to use content here, not a paragraph
If uri is not defined - you'll get error 500 on template level